src/app/containers/ArticleFigure/index.jsx

```diff
@@ -35,6 +35,7 @@ const ArticleFigure = ({
 }) => {
   const imageSpan = {
     default: '6',
+    group1: '6',
```

Comment:
Groups will inherit the `default` value, so setting `group1` to the same value is unnecessary. This change works, because portrait images have a _different_ default value. We should probably apply this rule within the control block on line 47 instead. E.G.:
```js
  if (height > width) {
    ParentWrapper = NestedGridParentSmall;
    ChildWrapper = NestedGridItemChildSmall;
    imageSpan.default = '4';
    imageSpan.group1 = '6';
  }
```

Reply:
Great suggestion. Thanks @simonsinclair